templates/deployments/appname/app.jsonnet.tpl

```diff
@@ -369,6 +389,40 @@ local all = {
 			},
 		},
 	},
+  {{- if (stencil.Arg "hpa.enabled") }}
+    hpa: ok.HorizontalPodAutoscaler(app.name, app.namespace) {
+      apiVersion: 'autoscaling/v2beta2',
+      target:: $.deployment,
+      spec+: {
+        minReplicas: currentHpaReplicaConfig.minReplicas,
```

Comment:
Would something like this `minRepliaces: if app.Environment == "staging" then hpaReplicasConfig.staging.minReplicas` work for both staging and production?

Reply:
this covers the devenv scenario too so its 2 if-else blocks and hence preferred a variable